app/components/forms/orders/guest-order-form.js

```diff
@@ -2,6 +2,8 @@ import Component from '@ember/component';
 import FormMixin from 'open-event-frontend/mixins/form';
 
 export default Component.extend(FormMixin, {
+  autoScrollToErrors: false,
```

Comment:
Making just change to one place , i dont think this is the right solution @uds5501 , I was thinking to remove it from complete app

Reply:
It still solves #2568 But there are many other too 😅 ,Still for #2568 its the right solution , Lets see what mentors review , 👍

Reply:
@kushthedude It is actually desirable to scroll up to that error. The problem in this particular case here is that this is a modal small enough to not require a scroll